src/app/components/wrapper-components/tutorial-game-wrapper/tutorial-game-wrapper.wrapper.component.spec.ts

```diff
@@ -811,9 +812,39 @@ describe('TutorialGameWrapperComponent (wrapper)', () => {
             // And the move done should have triggered the animation
             expect(testUtils.getGameComponent().updateBoard).toHaveBeenCalledWith(true);
         }));
+
+        it('should hide last move after click', fakeAsync(async() => {
+            // Given a TutorialStep for specific move
+            const tutorial: TutorialStep[] = [
+                TutorialStep.fromMove(
+                    'title',
+                    'Put your piece in a corner and give the opposite one.',
+                    QuartoRules.get().getInitialState(),
+                    [
+                        new QuartoMove(0, 0, QuartoPiece.BBBB),
+                        new QuartoMove(0, 3, QuartoPiece.BBBB),
+                        new QuartoMove(3, 3, QuartoPiece.BBBB),
+                        new QuartoMove(3, 0, QuartoPiece.BBBB),
+                    ],
+                    TutorialStepMessage.CONGRATULATIONS(),
+                    'Perdu.',
+                ),
+            ];
+            await wrapper.startTutorial(tutorial);
+            const gameComponent: AbstractGameComponent = testUtils.getGameComponent();
+            spyOn(gameComponent, 'hideLastMove').and.callThrough();
+
+            // When doing that click
```

Comment:
when doing that move ou when doing the first click, car sinon "that click" est indéfini